### DISCUSSION TITLE
[Q&A] LobeHub和ChatGPT-Next-Web之间，在手机上只能生成一个桌面图标？

### DISCUSSION THREAD

**Discussion by ccdanpian** [lobehub/lobehub#832] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

请问这是浏览器限制的，一个手机上只能有一个web app吗？ 还是 LobeHub和ChatGPT-Next-Web在某些方面用了同样的东西导致的。
有没有办法解决。

[a comment by an automated account is not shown]

**Comment by mushan0x0** [comment]
不能复现，请问是什么手机什么系统，用什么浏览器打开的

[a comment by an automated account is not shown]

**Comment by ccdanpian** [comment]
手机是小米13 pro，浏览器是chrome，chrome上已经没有安装选项了，只有打开lobe hub的选项。 firefox倒是有安装选项，点了没有反应。

[a comment by an automated account is not shown]

**Comment by ccdanpian** [comment]
试着用手机自带的浏览器，可以再生成一个桌面应用了。

[a comment by an automated account is not shown]

**Comment by canisminor1990** [comment, marked as answer]
可以通过右上角添加到主屏幕安装，我目前也是小米手机，有时因为网络问题请求 PWA 安装会比较慢，需要多等一会儿
